Resources/Prototypes/_Sunrise/borg_types.yml

```
@@ -40,6 +40,7 @@
   - type: InnateItem
     instantActions:
     - PortableSurveillanceCameraMonitor
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)